content/docs/releases/v1.0.0.md

```diff
@@ -0,0 +1,113 @@
+
+### Wireguard
+
+[Wireguard](https://www.wireguard.com/) tunnels are used in NSM v1.0.0 as the default transport
+type for IP payloads.
+
+# SRIOV
+
+[SRIOV](https://en.wikipedia.org/wiki/Single-root_input/output_virtualization) allows a single physical NIC to be 
+'shared' as it is multiple physical NICs.  NSM now support using either [vfio](https://www.kernel.org/doc/Documentation/vfio.txt) or
+a kernel interface backed by SRIOV as the mechanism by which a workload is connected to a Network Service.
+
+### Markdown Driven Testing
+
+Network Service Mesh documents its examples in [deployment-k8s](https://github.com/networkservicemesh/deployments-k8s) in Markdown.
+Network Service Mesh's testing is now generated by compiling that Markdown to Go tests in [integration-tests](https://github.com/networkservicemesh/integration-tests)
+
+### Topology Aware Scale from Zero
+
+When a workload requests a Network Service, the Network Service can specify topology preferences.
+For example the Network Service definition may define that the workload should be connected via a vWire to
+an Endpoint providing that Network Service on the same Node, or in the same Cluster, etc as the workload.
+
+In the absense of a topologically appropriate Endpoint for a workload, NSM can now spawn such an Endpoint.
+That Endpoint will retire itself after a period of idleness in which it has not been in use.
+
+### Latency Reduction
+
+All of the components of Network Service Mesh have gone through an aggressive reduction in their latency
+to reduce overall systemic latency between workload request and fulfilment of that request.
+This reduction also reduces the time it takes to reconverge in the event of the failure of one or more components.
+
+### Refactoring
+
+#### Simplification
+
+NSM has been simplified such that it only has two APIs:
+
+**NetworkService**
+: Used to Request, Close, or Monitor vWire Connections between a Client and and Endpoint providing the requested Network Service
+
+**Registry**
+: Used to Register, UnRegister, and Find Network Services and the Network Service Endpoints that provide them
+
+All sdk elements are written to provide a chain element implementing one of these APIs.
+Executable components are written by chaining together these chain elements in a chain of responsibility pattern.
+
+This vastly simplifies both writing new chain elements and consuming them to create new Network Service Endpoints, Forwarders, or other Components.
+
+#### Multi-repo
+Network Service Mesh has been refactored from a monorepo ([networkservicemesh/networkservicemesh](https://github.com/networkservicemesh/networkservicemesh)) into
+multi-repos:
+
+{{< object data="/img/multi-repo.svg" type="image/svg+xml" >}}
+
+**[api](https://github.com/networkservicemesh/api)**
+: Just the  NSM apis
+
+**[sdk](https://github.com/networkservicemesh/sdk)**
+: Platform independent sdk
+
+**[sdk-*]*(https://github.com/networkservicemesh?q=sdk-+is:repo)**
```

Comment:
Asterisk in the middle is probably a typo: `]*(`

Reply:
Fixed